clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -59,12 +59,27 @@ class ServerGroupController {
 
   @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ') and hasPermission(#account, 'ACCOUNT', 'READ')")
   @RequestMapping(value = "/applications/{application}/serverGroups/{account}/{region}/{name:.+}", method = RequestMethod.GET)
-  ServerGroup getServerGroup(@PathVariable String application, // needed for @PreAuthorize
+  ServerGroup getServerGroupByApplication(@PathVariable String application, // needed for @PreAuthorize
                              @PathVariable String account,
                              @PathVariable String region,
                              @PathVariable String name) {
+    getServerGroup(account, region, name)
+  }
+
+  @PreAuthorize("hasPermission(#account, 'ACCOUNT', 'READ')")
+  @PostAuthorize("hasPermission(returnObject?.moniker?.app, 'APPLICATION', 'READ')")
+  @RequestMapping(value = "/serverGroups/{account}/{region}/{name:.+}", method = RequestMethod.GET)
+  ServerGroup getServerGroupByName(@PathVariable String account,
```

Comment:
The distinction in method names between `getServerGroupByApplication` and `getServerGroupByName` is not clear to me. Perhaps `getServerGroupByMoniker`?